idd/Energy+.idd.in

```diff
@@ -73954,6 +73954,337 @@ HeatPump:PlantLoop:EIR:Heating,
+       \default 60
+       \units C
+       \note Design Supply Temperature in [degree C]
+  N5 , \field Design Temperature Lift
+       \autosizable
+       \default 11.1
+       \units deltaC
+       \note Design Temperature Lift in [degree C]
+  N6 , \field Sizing Factor
+       \minimum 1.0
+       \default 1.0
+       \note Sizing Factor for equipment sizing
+  A8 , \field Flow Mode
+       \required-field
+       \type choice
+       \key NotModulated
+       \key ConstantFlow
+       \key LeavingSetpointModulated
+       \default NotModulated
+       \note Flow Mode for the water side of the fuel-fired absorption heat pump
+  A9 , \field Outdoor Air Temperature Curve Input Variable
+       \required-field
+       \type choice
+       \key DryBulb
+       \key WetBulb
+       \default DryBulb
+       \note Outdoor air temperature curve input variable;
+       \note The options are Outdoor Air Dry Bulb or Wet Bulb temperature for curves
+  A10, \field Water Temperature Curve Input Variable
+       \required-field
+       \type choice
+       \key EnteringCondenser
+       \key LeavingCondenser
+       \default EnteringCondenser
+       \note Water Temperature curve input variable - Condenser Entering or Leaving Water Temperature for curves
+  A11, \field Normalized Capacity Function of Temperature Curve Name
+       \required-field
+       \type object-list
+       \object-list BivariateFunctions
+       \note: CAPFT - Normalized Capacity Function of Temperature Curve name,
+       \note which is a biquadratic curve or a lookup table.
+  A12, \field Fuel Energy Input Ratio Function of Temperature Curve Name
+       \required-field
+       \type object-list
+       \object-list BivariateFunctions
+       \note EIRFT - Fuel Energy Input Ratio Function of Temperature Curve Name,
+       \note which is a biquadratic curve or a lookup table.
+  A13, \field Fuel Energy Input Ratio Function of PLR Curve Name
+       \required-field
+       \type object-list
+       \object-list UnivariateFunctions
+       \note EIRFPLR - Fuel Energy Input Ratio Function of Part Load Ratio(PLR) Curve Name,
+       \note which is a cubic curve or a lookup table.
+  N7 , \field Minimum Part Load Ratio
+       \minimum> 0.0
+       \maximum 1.0
+       \default 0.1
+       \note Minimum Part Load Ratio (PLR) in between 0 and 1
+  N8 , \field Maximum Part Load Ratio
+       \minimum> 0.0
+       \maximum 1.0
+       \default 1.0
+       \note Maximum Part Load Ratio (PLR) in between 0 and 1
+  A14, \field Defrost Control Type
+       \type choice
+       \key Timed
+       \key OnDemand
+       \default Timed
+       \note Defrost operation control type: Timed or OnDemand
+  N9 , \field Defrost Operation Time Fraction
+       \minimum 0
+       \maximum 1
+       \default 0
+       \note Defrost operation time fraction, which will be used for timed defrost control type.
+  A15, \field Fuel Energy Input Ratio Defrost Adjustment Curve Name
```

Comment:
But this curve needs a note that it's only used for OnDemand defrost.

Reply:
The note is added now.